lib/btc/BtcRpc.js

```diff
@@ -45,15 +45,36 @@ class BtcRpc {
     return this.asyncCall('cmdlineUnlock', [time]);
   }
 
+  async sendMany({ obj, options }) {
+    for (let tx in obj) {
+      obj[tx] /= 1e8;
+    }
+    const array = ['', obj];
+    if (options) {
+      array.push(options);
+    }
+    return this.asyncCall('sendMany', array);
+  }
+
   async sendToAddress({ address, amount }) {
     return this.asyncCall('sendToAddress', [address, amount / 1e8]);
   }
 
+  async unlockAndSendManyBatched({ obj, passphrase, options, time = 10800 }) {
```

Comment:
So I actually want to change this to accept an array of already batched inputs, and then iterate and call sendMany on those inputs